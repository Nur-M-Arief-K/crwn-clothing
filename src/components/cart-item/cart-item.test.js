import { render } from "@testing-library/react";
import CartItem from "./cart-item.component";

describe("CartItem tests", () => {
    test("it should render correct information of given props", () => {
        const cartItemMock = {
                id: 1,
                name: "cool hat",
                price: 2,
                imageUrl: "https://mock-url.com/",
                quantity: 1,
        };

        expect(render(<CartItem cartItem={cartItemMock} />)).toMatchSnapshot();
    });
});