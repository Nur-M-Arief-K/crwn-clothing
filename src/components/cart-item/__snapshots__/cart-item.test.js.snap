// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CartItem tests it should render correct information of given props 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="sc-bcXHqe hrTwlT"
      >
        <img
          alt="cool hat"
          src="https://mock-url.com/"
        />
        <div
          class="sc-gswNZR ivYAqT"
        >
          <span
            class="name"
          >
            cool hat
          </span>
          <span
            class="price"
          >
            1
             x 
            2
          </span>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="sc-bcXHqe hrTwlT"
    >
      <img
        alt="cool hat"
        src="https://mock-url.com/"
      />
      <div
        class="sc-gswNZR ivYAqT"
      >
        <span
          class="name"
        >
          cool hat
        </span>
        <span
          class="price"
        >
          1
           x 
          2
        </span>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
